libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/targets/js/npm/Npm.kt

```diff
@@ -141,12 +141,12 @@ class Npm internal constructor(
     ) {
         val progressLogger = objects.newBuildOpLogger()
         execWithProgress(progressLogger, description, execOps = execOps) { execSpec ->
-            val arguments: List<String> = mutableListOf<String>().apply {
+            val arguments = buildList {
```

Comment:
This changes are quite unrelated to what I generally do in this PR, so I can either revert them or submit a different PR, or leave them